docs/setup/settings.asciidoc

```diff
@@ -5,49 +5,65 @@ The Kibana server reads properties from the `kibana.yml` file on startup. The de
 Elasticsearch. This value must be a positive integer.
+
 `elasticsearch.shardTimeout:`:: *Default: 0* Time in milliseconds for Elasticsearch to wait for responses from shards. Set to 0 to disable.
+
 `elasticsearch.ssl.certificate:` and `elasticsearch.ssl.key:`:: Optional settings that provide the paths to the PEM-format SSL
 certificate and key files. These files are used to verify the identity of Kibana to Elasticsearch and are required when `xpack.ssl.verification_mode` in Elasticsearch is set to either `certificate` or `full`.
-`elasticsearch.ssl.certificateAuthorities:`:: Optional setting that enables you to specify a list of paths to the PEM file for the certificate
-authority for your Elasticsearch instance.
+
+`elasticsearch.ssl.certificateAuthorities:`:: Optional setting that enables you to specify a list of paths to the PEM file for the certificate authority for your Elasticsearch instance.
+
 `elasticsearch.ssl.keyPassphrase:`:: The passphrase that will be used to decrypt the private key. This value is optional as the key may not be encrypted.
+
 `elasticsearch.ssl.verificationMode:`:: *Default: full* Controls the verification of certificates presented by Elasticsearch. Valid values are `none`, `certificate`, and `full`.
 `full` performs hostname verification, and `certificate` does not.
+
 `elasticsearch.startupTimeout:`:: *Default: 5000* Time in milliseconds to wait for Elasticsearch at Kibana startup before
 retrying.
+
 `elasticsearch.url:`:: *Default: "http://localhost:9200"* The URL of the Elasticsearch instance to use for all your
 queries.
+
 `elasticsearch.username:` and `elasticsearch.password:`:: If your Elasticsearch is protected with basic authentication,
 these settings provide the username and password that the Kibana server uses to perform maintenance on the Kibana index at
 startup. Your Kibana users still need to authenticate with Elasticsearch, which is proxied through the Kibana server.
 
 `kibana.defaultAppId:`:: *Default: "discover"* The default application to load.
+
 `kibana.index:`:: *Default: ".kibana"* Kibana uses an index in Elasticsearch to store saved searches, visualizations and
 dashboards. Kibana creates a new index if the index doesn’t already exist.
 
 `logging.dest:`:: *Default: `stdout`* Enables you specify a file where Kibana stores log output.
+
 `logging.quiet:`:: *Default: false* Set the value of this setting to `true` to suppress all logging output other than
 error messages.
+
 `logging.silent:`:: *Default: false* Set the value of this setting to `true` to suppress all logging output.
-`logging.verbose:`:: *Default: false* Set the value of this setting to `true` to log all events, including system usage
-information and all requests.
+
+[[logging-verbose]]`logging.verbose:`:: *Default: false* Set the value of this setting to `true` to log all events, including system usage information and all requests. Supported on Elastic Cloud Enterprise.
```

Comment:
Wondering why this one is surrounded in square brackets while the others above aren't... What exactly does this do?

Reply:
This is the only setting that is supported in ECE (for logging). These brackets, when converted, create an HTML anchor that I'm linking to from the Cloud docs.

Reply:
:+1: